recipes/SimpleElastix/bld.bat

```diff
@@ -0,0 +1,7 @@
+mkdir build
+cd build
+cmake ../SuperBuild -G "%CMAKE_GENERATOR%"
```

Comment:
[Here's an example](https://github.com/conda-forge/opencv-feedstock/blob/master/recipe/bld.bat#L47). You can also avoid invoking nmake directly by using cmake like [so](https://github.com/conda-forge/opencv-feedstock/blob/master/recipe/bld.bat#L103).